packages/flutter/lib/src/material/adaptive_text_selection_toolbar.dart

```diff
@@ -242,9 +248,8 @@ class AdaptiveTextSelectionToolbar extends StatelessWidget {
     switch (Theme.of(context).platform) {
       case TargetPlatform.iOS:
         return buttonItems.map((ContextMenuButtonItem buttonItem) {
-            return CupertinoTextSelectionToolbarButton.text(
-              onPressed: buttonItem.onPressed,
-              text: getButtonLabel(context, buttonItem),
+            return CupertinoTextSelectionToolbarButton.buttonItem(
+              buttonItem: buttonItem.copyWith(label: getButtonLabel(context, buttonItem)),
```

Comment:
Oh.Before I see `CupertinoTextSelectionToolbarButton.text` and add a text for it. To keep that logic the same, I did that. But now it seems can also work well with `CupertinoTextSelectionToolbarButton.buttonItem` since the text will get from `CupertinoTextSelectionToolbarButton.getButtonLabel`. So result should be the same. Changed^_^.